specs/IR-0006/suite/test_spec_ir0006.py

```
@@ -0,0 +1,174 @@
+
+    def test_nongoals(self):
+        fm, _ = _fm_text()
+        items = [l for l in _fm_list_block(fm, "nonGoals").splitlines() if l.startswith("- ")]
+        self.assertGreaterEqual(len(items), 5, "nonGoals 过少")
+
+    def test_blastradius(self):
+        fm, _ = _fm_text()
+        items = [l for l in _fm_list_block(fm, "blastRadius").splitlines()
+                 if l.startswith("- repo:")]
+        self.assertGreaterEqual(len(items), 3, "blastRadius 仓面过少")
+        for l in items:
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)